.circleci/config.yml

```diff
@@ -431,40 +431,45 @@ workflows:
           requires:
             - approve/prisma-migrate
 
-  main:
+  terraform-trigger:
     when: << pipeline.parameters.workflow_main >>
     jobs:
       - terraform/trigger:
           docker: *orb_docker_terraform
-          name: terraform/trigger
+          name: terraform/trigger/dev
           system: realtime-streaming
           path: .
+      - terraform/trigger:
+          docker: *orb_docker_terraform
+          name: terraform/trigger/prd
+          system: realtime-streaming
```

Comment:
Are we planning to deploy using realtime-streaming system or llm-ops system?

Reply:
realtime-streaming system